index.js

```diff
@@ -0,0 +1,73 @@
+const cl = console.log;
+const express = require("express");
+
+const db = require("./middleware/helpers");
+const PORT = 4700;
+const server = express();
+
+server.use(express.json());
+
+server.get("/notes", (req, res) => {
+  db.getNotes()
+    .then(notes => {
+      res.json(notes);
+    })
+    .catch(err => {
+      res.status(500).send(err);
+    });
+});
+
+server.get("/notes/:id", (req, res) => {
+  const { id } = req.params;
+  db.getNotes(id)
+    .then(note => {
+      res.json(note);
+    })
+    .catch(err => {
+      res.status(500).send(err);
+    });
+});
+
+server.post("/notes/create", (req, res) => {
+  const note = req.body;
+  db.addNote(note)
+    .then(ids => {
+      res.status(201).json(ids[0]);
```

Comment:
This is more semantic https://knexjs.org/#Builder-first

Reply:
I literally used it on the helper function right before it. I have no idea why I didn't there.